templates/_includes/photos_footer.html

```diff
@@ -0,0 +1,27 @@
+{% from '_includes/_defaults.html' import PHOTOS_LIGHTBOX with context %}
```

Comment:
Does this work? I have a feeling it won't work.

See how we do it with disqus script

https://github.com/Pelican-Elegant/elegant/blob/master/templates/_includes/disqus_scripts.html
https://github.com/Pelican-Elegant/elegant/blob/master/templates/_includes/comments.html

Reply:
It does with the current approach, rendered html contains the block and works fine